docs/en/sim_gazebo_gz/plugins.md

```diff
@@ -0,0 +1,79 @@
+# Gazebo Plugins
+
+Gazebo plugins extend the simulator with custom functionality not provided by default. They can be attached to different entity types and allow you to add new sensors, modify world physics, or interact with the simulation environment.
+
+## Plugin Types
+
+Plugins can be attached to these entity types:
+
+- **World** - Global simulation behavior
+- **Model** - Specific model functionality
+- **Sensor** - Custom sensor implementations
+- **Actor** - Dynamic entity behavior
+
+## Supported Plugins
+
+PX4 currently supports these plugins:
+
+- [OpticalFlowSystem](https://github.com/PX4/PX4-Autopilot/tree/main/src/modules/simulation/gz_plugins/optical_flow): Provides optical flow sensor simulation using OpenCV-based flow calculation.
+- [GstCameraSystem](https://github.com/PX4/PX4-Autopilot/tree/main/src/modules/simulation/gz_plugins/gstreamer): Streams camera feeds via UDP (RTP/H.264) or RTMP with optional NVIDIA CUDA hardware acceleration.
+- [MovingPlatformController](https://github.com/PX4/PX4-Autopilot/tree/main/src/modules/simulation/gz_plugins/moving_platform_controller): Controls moving platforms (ships, trucks, etc.) for takeoff and landing scenarios.
+  Includes configurable velocity, heading, and random fluctuations.
```

Comment:
FYI, I changed like this for the links - makes it easier for people to find the sources, and for me to know there is a change if something moves.